### PR TITLE
NMS-15100: Change needed to a file name reference in documentation

### PR DESCRIPTION
**Pull request by Bonrob2**
NMS-15100: Change needed to a file name reference in documentation - updated the name of the file that needs to be moved as part of the TSDB set up.
NMS-14820: Documentation doesn't render asciimath properly: removed the 3 formulas that were listed. They did not add any value, and the work involved to add an asciimath renderer to the environment outweighed any benefit.

### All Contributors

* [ ] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-15100
* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14820

### CODE REVIEW

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 1]
```diff
@@ -30,7 +30,7 @@ However, you need to move the configuration files into the correct location and
 
 [source, console]
 ----
-cp /opt/opennms/etc/examples/opennms.properties.d/timeseries.properties /opt/opennms/etc/opennms.properties.d/timeseries.properties
+cp /opt/opennms/etc/examples/opennms.properties.d/hosted_tsdb.timeseries.properties
```
Comment (mmahacek) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:33]: The command is invalid. There should be a source and destination file, but the change combined the two.

Comment (Bonrob2) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:33]:
```suggestion
cp /opt/opennms/etc/examples/opennms.properties.d/hosted_tsdb.timeseries.properties /opt/opennms/etc/opennms.properties.d/timeseries.properties
```

Comment (Bonrob2) [reply docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:33]: @mmahacek is this correct?

**Review by mmahacek — APPROVED** [review]
LGTM

**Review by emarsh1995 — APPROVED** [review]
LGTM. Thank you!

**Review by drewym — COMMENTED** [review]
Yes, this now looks correct.

**Pull request merged**